src/bookshelf/stest/rebar.config

```diff
@@ -20,6 +21,6 @@
 %% markdown can be checked in and will be browsable on github. The
```

Comment:
Can we remove this block entirely, since it's for stest and I edown isn't a thing in our tests?